README.md

```diff
@@ -13,26 +13,35 @@ matrices that have uncertain parameters.
 
 - [Manual](http://juliareach.github.io/IntervalMatrices.jl/latest/)
 - [Contributing](https://juliareach.github.io/IntervalMatrices.jl/latest/about/#Contributing-1)
+- [Release notes of tagged versions](https://github.com/JuliaReach/IntervalMatrices.jl/releases)
+- [Release notes of the development version](https://github.com/JuliaReach/IntervalMatrices.jl/wiki/Release-log-tracker)
+- [Publications](https://juliareach.github.io/Reachability.jl/latest/publications/)
```

Comment:
should we keep this? as none of the "Publications" listed there uses `IntervalMatrices`.

Reply:
I did not know the policy. I thought that we link there from every `JuliaReach` repository. And in the future this might change. But I can remove it again.